recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - boost-histogram >=1.0.0
     - awkward >=1.8
     - tabulate >=0.8.1
-    - matplotlib-base
+    - matplotlib-base >=3.5.0
```

Comment:
`cabinetry` 0.5.1 added a lower bound to `matplotlib` (see [diff](https://github.com/scikit-hep/cabinetry/compare/v0.5.0...v0.5.1#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52)), propagating this here.